eap-aro/src/main/scripts/jboss-setup.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-set -Eeuo pipefail
```

Comment:
Remove this, as it will cause some unexpected errors and kill the container environment running the jboss-setup.sh.